### PR TITLE
[timer@Severga] Allow translations - Fix sound playback

### PR DESCRIPTION
**Pull request by claudiux**
@Severga @brownsr @mtwebster 

The purpose of this PR is to offer some improvements to the timer@Severga applet:

  * Allow translations: In the `po/` folder, I created the `makepot` script used to build the `timer@Severga.pot` file. I used this .pot file to create the fr.po file (French translation).
  * I fixed sound playback, because `global.play_sound_file()` is not a function (error notified in `~/.xsession-errors` - Cinnamon 5.6.7).
  * The `applet.js` script was modified to use translations and the SoundManager.playSoundFile() function.

Using makepot:
  * `cd ~/.local/share/cinnamon/applets/timer@Severga/po`
  * `./makepot` to create timer@Severga.pot file
  * Use timer@Severga.pot with poEdit to make translations
  * `./makepot -i` to install translations
  * Restart Cinnamon.

### CODE REVIEW

**Comment by Severga** [comment]
I've made a Russian localization. You could add it too.
[ru_RU.po.zip](https://github.com/linuxmint/cinnamon-spices-applets/files/10826039/ru_RU.po.zip)
Hope it will look OK.

**Comment by claudiux** [comment]
@Severga Your .po file has been added.

File: timer@Severga/files/timer@Severga/applet.js [hunk 1]
```diff
@@ -12,530 +12,551 @@ const Mainloop = imports.mainloop;
...
+  on_sminus_clicked: function() {
+    if (this.sSpin.value > 0) this.sSpin.value--;
+    else this.sSpin.value = 59;
+    this.sSpin.text = (this.sSpin.value < 10 ? "0" : "") + this.sSpin.value;
+  },
+
+  on_applet_clicked: function(event) {
+    this._remove_flashTimeout();
+    this.actor.set_style("");
+    if (this.openSubmenu == "Quick setup") this.quickMenuItem.menu.open();
```
Comment (Severga) [inline timer@Severga/files/timer@Severga/applet.js:375]: Does this work right with localization?

Comment (claudiux) [reply timer@Severga/files/timer@Severga/applet.js:375]: @Severga 
Yes, it works. It doesn't work if you translate it. This is the subject of this commit: https://github.com/linuxmint/cinnamon-spices-applets/pull/4688/commits/0a22e9d5d007e212d04d91a617ca0b471cdaa28f

**Comment by claudiux** [comment]
@Severga 
Did you try your .po file?
To do that:
```
cd ~/.local/share/cinnamon/applets/timer@Severga/po
./makepot -i
```
Then, restart Cinnamon.
If translations do not appear, then try to rename your file from `ru_RU.po` to `ru.po` and again:
`./makepot -i` then restart Cinnamon.

Please tell me if everything is fine (and in which of these cases).

**Comment by Severga** [comment]
@claudiux, please, give me your variant of the applet. I don't know how to get it from here.

**Comment by Severga** [comment]
@claudiux, I tested my po-file having renamed it into `en.po`. It looks OK. I can't check whether `ru_RU.po` and `ru.po` work fine, because I use only English localization on my Mint. I think it would be better to make both of these files with the same content. But in other applets I found only `ru.po` files.
By the way, as a suggestion, I think for the French translation it would look better if "In" and "At" were of the same size and right-justified:
`"Dans:"`
`"   À:"`

**Comment by claudiux** [comment]
@Severga 
I made the necessary changes.

**Comment by Severga** [comment]
@claudiux
Thanks for your work!

**Comment by claudiux** [comment]
@brownsr @mtwebster 
This PR is now ready. Can you merge, please?
Regards.
Claudiux

**Pull request merged**